tests/queryEditor.page.ts

```diff
@@ -0,0 +1,74 @@
+import { type Page, type Locator } from '@playwright/test';
+import { type PanelEditPage } from '@grafana/plugin-e2e';
+
+/**
+ * Query Editor page object model testing utility providing properties and
+ * methods for common locators and actions.
+ */
+export class QueryEditorPage {
```

Comment:
```suggestion
export class QueryEditor {
```

Minor nit: can this be named `QueryEditor` since it's a component within the `PanelEditPage`. It would be good to name this file `queryEditor.model.ts` along with this change.

Reply:
Done!